spec/support/table_row_matcher.rb

```diff
@@ -0,0 +1,51 @@
+module TableRowMatchers
+  class SimpleTableRowMatcher < Capybara::RSpecMatchers::Matcher
```

Comment:
http://www.relishapp.com/rspec/rspec-expectations/v/3-4/docs/custom-matchers is a good starting point, the reason this doesn't use the short-hand style

```ruby
RSpec::Matchers.define :be_a_multiple_of do |expected|
  match do |actual|
    actual % expected == 0
  end
end
```

is that I wanted it to work with capybara page objects and that requires a bit more effort.  Happily there's a Capybara rspec matcher base class to use and we can take "inspiration" from the existing matchers that Capybara provides (see: https://github.com/jnicklas/capybara/blob/master/lib/capybara/rspec/matchers.rb).

In general defining your own matchers for complex expectations can be quite powerful as it makes the specs read much nicer.  I'm often guilty of forgetting that this is possible or desirable though.  Only remembered this time because it was so verbose without writing a custom matcher.